app/views/exam_templates/assign_errors.html.erb

```diff
@@ -0,0 +1,279 @@
+                </div>
+                <div class="separator"></div>
+                <div class="row">
+                  <span data-l10n-id="document_properties_producer">PDF Producer:</span> <p id="producerField">-</p>
+                </div>
+                <div class="row">
+                  <span data-l10n-id="document_properties_version">PDF Version:</span> <p id="versionField">-</p>
+                </div>
+                <div class="row">
+                  <span data-l10n-id="document_properties_page_count">Page Count:</span> <p id="pageCountField">-</p>
+                </div>
+                <div class="buttonRow">
+                  <button id="documentPropertiesClose" class="overlayButton"><span data-l10n-id="document_properties_close">Close</span></button>
+                </div>
+              </div>
+            </div>
+          </div>  <!-- overlayContainer -->
+        </div>
+      </div>
+    </div>
+    <script>
+      function rotate() {
+        PDFView.rotatePages(90);
+      }
+      PDFView.onLoadComplete = function() {
+        annotation_manager = new PdfAnnotationManager(PDFView, 'viewer');
+        annotation_manager.resetAngle();
+      };
+      var index = 0;
+      var files = <%= raw @error_files %>;
+
+      function getFile(filename) {
+        $.ajax({
+          method: "GET",
+          url: "download_error_path",
+          dataType: "text",
+          data: {
+            assignment_id: <%= @assignment.id %>,
+            file_name: filename
+          },
+          success: function(data){
+            webViewerLoad(data);
+          },
+          error: function(err){
+            console.log(err);
+          }
+        });
+      }
+
+      function nextFile() {
+        if(index < files.length - 1){
+          index += 1;
+        }else{
+          index = 0;
+        }
+        getFile(files[index]);
+      }
+
+      function previousFile() {
+        if(index > 0){
+          index -= 1;
+        }else{
+          index = files.length - 1;
+        }
+        getFile(files[index]);
+      }
+      getFile(files[0]);
+
+    </script>
+
+    <div id='drag'></div>
+
+    <div id='right-pane' class='input_pane'>
+      <div class='ui-tabs-panel flex-col'>
+        <div class="side_header">
+          <h2>Document Information</h2>
+        </div>
+        <form id="assign_error" class="side_header">
+          <div class="side_header">
+            <label class="form_label" for="exam_template">Exam Template: </label>
+            <select id="exam_template">
+              <!--<option disabled selected>Select Template</option>-->
+              <% ExamTemplate.where(assignment_id: @assignment.id).each do |e| %>
+                  <option><%= e.name %></option>
+              <% end %>
+            </select>
+          </div>
+          <br>
+          <input id="copy" name="names" type="text" placeholder="Copy Number">
```

Comment:
The `name` attribute should be more meaningful. Also, it would be good to have a label tag for each of these. The placeholder text is long compared to the space that is required for the actual numbers.